storm-webapp/src/main/java/org/apache/storm/daemon/logviewer/webapp/LogviewerResource.java

```diff
@@ -137,7 +137,7 @@ public Response searchLogs(@Context HttpServletRequest request) throws IOExcepti
         String user = httpCredsHandler.getUserName(request);
         String topologyId = request.getParameter("topoId");
         String portStr = request.getParameter("port");
-        String callback = request.getParameter("callback");
+        String callback = request.getParameter("callbackParameterName");
```

Comment:
So why are we changing `/searchLogs?callback=bar` to `/searchLogs?callbackParameterName=bar`?  If that is what we are doing it is a regression, if not I am really missing something.